dev/dev/devDeployment.yaml

```diff
@@ -22,7 +22,8 @@ datarepo-api:
   image:
     tag: 9664c8a-develop
   env:
-    GOOGLE_PROJECTID: broad-jade-dev
+    GOOGLE_CLOUD_PROJECT: broad-jade-dev
+    GOOGLE_CLOUD_DATA_PROJECT: broad-jade-dev-data
```

Comment:
Actually, I think it needs to be this:
 GOOGLE_PROJECTID: broad-jade-dev
 GOOGLE_SINGLEDATAPROJECTID: broad-jade-dev-data